### ISSUE TITLE
Models are safetensor but this cannot run safetensor models.

### ISSUE THREAD

**Issue by Delcos** [harubaru/convogpt#4]
Safetensors don't support transformers, nor the tokenizer or the CausalLM which are used.

**Comment by harubaru** [comment]
Have you installed the safetensors dependency?

**Comment by Delcos** [comment]
Yeah, the script can't use **from transformers**, it needs to use safetensor instead. Or it could check for safetensor models and if non are found default to pytorch models, but as of now when running it gives an error about not finding any models.

**Comment by Delcos** [comment]
> Yeah, the script can't use **from transformers**, it needs to use safetensor instead. Or it could check for safetensor models and if non are found default to pytorch models, but as of now when running it gives an error about not finding any models.

The model itself is really interesting though, I know it's not the main focus but a simple model card on Huggingface would be good too. To know what training data was used.

**Comment by harubaru** [comment]
model card will be written when i finish training the base models
